src/metadata/metadata_superblock.c

```diff
@@ -230,9 +335,11 @@ struct ocf_pipeline_properties ocf_metadata_load_sb_pipeline_props = {
 	.steps = {
 		OCF_PL_STEP_FOREACH(ocf_metadata_store_segment,
 				ocf_metadata_load_sb_store_segment_args),
+		OCF_PL_STEP_ARG_INT(ocf_metadata_load_segment,
```

Comment:
analogous changes need to be applied to ocf_metadata_load_sb_recov_pipeline_props